.gitignore

```
@@ -13,6 +13,7 @@ data/flightjson
 data/nycflights
 data/myfile.zarr
 data/accounts.parquet
+**/dask-worker-space
```

Minor comment: use `dask-worker-space/` instead of  `**/dask-worker-space` for consistency with how we already `.gitignore` other directories

```suggestion
dask-worker-space/
```